locale/fi.po

```diff
@@ -2,12 +2,12 @@
 # Copyright (C) 2020 ORGANIZATION
 # This file is distributed under the same license as the PROJECT project.
 # Automatically generated, 2020.
-#
+# 
```

Comment:
random ghost spaces got added in some of the translations

Reply:
I'm going to make a CI check against this once I get back to work on Thrive after Christmas.

Reply:
I have no idea why this is happening.
Everything (apt, pip, gem) is up to date.
I'm using WSL(ubuntu).

Reply:
Was it @Kasterisk or @icedjuro who also got the same thing?
In the end the solution was to just manually go to the files and remove that space in order to get rid of it.

Reply:
I don't recall getting that extra spaces, I think you might be referring to Fuja's previous PR which also has that https://github.com/Revolutionary-Games/Thrive/pull/1950#discussion_r546406050. Not sure what's causing it.

Reply:
In case somebody else experiences this:
```unix
apt-get update
apt-get install dos2unix
```

then add this file to your `.git/hooks/pre-commit` file (create if it does not exist)
```
#!/bin/sh
cd locale/
find . -name '*.po' -type f -exec dos2unix {} \;
sed -i 's/# $/#/g' *
```